src/main/java/dev/jbang/cli/BaseScriptDepsCommand.java

```diff
@@ -0,0 +1,15 @@
+package dev.jbang.cli;
+
+import java.util.List;
+
+import picocli.CommandLine;
+
+public abstract class BaseScriptDepsCommand extends BaseScriptCommand {
+
+	@CommandLine.Option(names = { "--deps" }, description = "Add additional dependencies.")
```

Comment:
fine for now doing it by inheritance but I think picocli has notion of compositional reuse which might be better in long run...but ok for this pr to do it this way

Reply:
Yeah, it does, but it doesn´t make the code nicer, because in the end Java doesn't have a notion of "mixins", so suddenly you introduce an indirection everywhere you need to use those fields. (Now, indirections aren't bad or anything, but it just feels weird to me that among all the options we have suddenly there are 2 that need indirection).

But there might indeed come a point where just more levels of inheritance won't work, we'll think about what to do when we get there :-)

Reply:
yes, for this one its not a big win - but i think we are on the cusp where having mixins for everything feeding into "run" would be nice to get applied uniformly across all the commands.

Reply:
Yeah, you're probably right